Jenkinsfile

```diff
@@ -1,6 +1,4 @@
 // TODO: Come up with a way to enable binary generation on Linux but not
 // on Windows without needing to run buildPlugin multiple times to avoid
 // https://github.com/jenkinsci/durable-task-plugin/pull/126#issuecomment-681964838.
-withEnv(['SKIP_DURABLE_TASK_BINARY_GENERATION=true']) {
-  buildPlugin(platforms: ['docker', 'maven-windows'])
-}
+buildPlugin(platforms: ['docker', 'maven-windows'])
```

Comment:
```suggestion
buildPlugin(platforms: ['docker', 'maven-windows'])
```

(trailing newline)

Reply:
fixed in 388a7bf1c9a1ba7c8e00f1b367c01daaa95b1c0a